packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -120,31 +124,106 @@ describe('RPC Server Acceptance Tests', function() {
+            }
+
+        });
+
+        it('should execute "eth_getBlockByHash", hydrated transactions not specified', async function() {
+            const blockResult = await relay.call('eth_getBlockByHash', [mirrorBlock.hash]);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, false);
+        });
+
+        it('should execute "eth_getBlockByHash", hydrated transactions = false', async function() {
+            const blockResult = await relay.call('eth_getBlockByHash', [mirrorBlock.hash, false]);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, false);
+        });
+
+        it('should execute "eth_getBlockByHash", hydrated transactions = true', async function() {
+            const blockResult = await relay.call('eth_getBlockByHash', [mirrorBlock.hash, true]);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, true);
         });
 
-        it('should execute "eth_getBlockByHash"', async function() {
-            const relayBlock = await relay.call('eth_getBlockByHash', [mirrorBlock.hash, 'true']);
-            Assertions.block(relayBlock, mirrorBlock);
+        it('should execute "eth_getBlockByHash" for non-existing hash, hydrated transactions = true', async function() {
+            const blockResult = await relay.call('eth_getBlockByHash', [NON_EXISTING_BLOCK_HASH, true]);
+            expect(blockResult).to.be.null;
         });
 
-        it('should execute "eth_getBlockByNumber"', async function() {
+        it('should execute "eth_getBlockByHash" for non-existing hash, hydrated transactions = false', async function() {
+            const blockResult = await relay.call('eth_getBlockByHash', [NON_EXISTING_BLOCK_HASH, false]);
+            expect(blockResult).to.be.null;
+        });
+
+        it('should execute "eth_getBlockByNumber", hydrated transactions not specified', async function() {
+            const blockResult = await relay.call('eth_getBlockByNumber', [mirrorBlock.number]);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, false);
+        });
+
+        it('should execute "eth_getBlockByNumber", hydrated transactions = false', async function() {
+            const blockResult = await relay.call('eth_getBlockByNumber', [mirrorBlock.number, false]);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, false);
+        });
+
+        it('should execute "eth_getBlockByNumber", hydrated transactions = true', async function() {
             const blockResult = await relay.call('eth_getBlockByNumber', [mirrorBlock.number, true]);
-            Assertions.block(blockResult, mirrorBlock);
+            Assertions.block(blockResult, mirrorBlock, mirrorTransactions, true);
+        });
+
+        it('should execute "eth_getBlockByNumber", hydrated transactions = true', async function() {
```

Comment:
Updated test names to make sure they are all unique and describe the unique test